README.md

```diff
@@ -25,38 +25,39 @@ Then you have to choose between the different configs, depending on the project
   - react
   - react-native
 
-You'll also need to include eslint & any other plugins/parsers that are needed in your package.json.
-Particularly:
+You'll also need to include eslint & any other plugins/parsers that are needed in your package.json. Particularly:
```

Comment:
Not necessary in node 5+. Perhaps this line should say that it should be a dependency only if node 4 or less.